modules/virt-configuring-certificate-rotation.adoc

```diff
@@ -0,0 +1,42 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-certificate-rotation.adoc
+
+[id="virt-configuring-certificate-rotation_{context}"]
+
+Configure certificate rotation parameters to replace existing certificates. You can do this during OpenShift Virtualization installation in the web console or after installation in the YAML file.
+
+.Procedure
+
+. Open the `HyperConverged` Custom Resource (CR) by running the following command:
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
+
+. Edit the `spec.certificateRotation` fields below. All values must be 10 minutes in length or greater to avoid overloading the system, and should be expressed as strings that comply with golang's ParseDuration format (https://golang.org/pkg/time/#ParseDuration):
+
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+ name: kubevirt-hyperconverged
+ namespace: openshift-cnv
+spec:
+  certificateRotation:
```

Comment:
The name of the object is `certConfig`, not `certificateRotation`.
Reference:
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/a515416f99542501a841d5e0bdf6bcd320adc9b6/pkg/apis/hco/v1beta1/hyperconverged_types.go#L58
https://github.com/kubevirt/hyperconverged-cluster-operator/blob/release-1.4/docs/api.md#hyperconvergedspec